dali/pipeline/graph/graph_descr.cc

```diff
@@ -162,11 +162,8 @@ void OpGraph::AddOp(const OpSpec &spec, const std::string &op_name) {
     new_node.parent_tensors.push_back(consumed_tensor_id);
 
     // Update the consumer info for this tensor
-    TensorMeta meta;
-    meta.node = new_node.id;
-    meta.index = i;
-    meta.is_support = spec.IsArgumentInput(i);
-    meta.storage_device = ParseStorageDevice(spec.InputDevice(i));
+    TensorMeta meta = {new_node.id, i, ParseStorageDevice(spec.InputDevice(i)),
```

Comment:
I liked think the explicit field assigmnent was better - this will break (possibly silently) when we add fields to TensorMeta.

Reply:
Done